coal/mineral.py

```diff
@@ -6,20 +6,25 @@
 # define number of layers in normalized pixel
 NUMBER_OF_LAYERS = 128
 
-# define wavelength range
-WAVELENGTH_MIN = ???
-WAVELENGTH_MAX = ???
+# define wavelength range in nanometers
+WAVELENGTH_MIN = 383.150
+WAVELENGTH_MAX = 2508.200
 
 def normalize(image):
-
     """
-    Return a copy of an MxNxB image (or 1x1xB pixel) with constrained wavelength range and normalized spectra.
+    This takes the file name of the library to be used in the training and returns a
+    classifier
+    Args:
+       image (str): the image to normalize
+    Returns:
+       normalizedImage (LinearTransform): LinearTransform object?
```

Comment:
it looks like the comment got duplicated with the one below.

i expected that normalize would take an image/pixel/ndarray and return the normalized version as an image/pixel/ndarray. that is, `normalize` is a function from `ndarray P` to `ndarray Q` such that `P.shape == (M, N, B)` and `Q.shape == (M, N, NUMBER_OF_LAYERS)`.

note that using `ndarray` makes this function more general than its subclass `SpyImage`, allowing us to use it to normalize both individual pixels as well as entire spectral libraries.